server/src/resources/cart-item/cart-item.repository.ts

```diff
@@ -0,0 +1,67 @@
+import { defaultCartItemDb } from '../../constants';
+import { CartItemDb } from '../../database/models.db';
+import { ICartItem, CartItemEntity } from './cart-item.interfaces';
+
+class CartItemRepository {
+  constructor() {
+    const addDefaultCartItems = async () => {
+      setTimeout(() => {
+        Promise.all(defaultCartItemDb.map(async (cartItem) => {
+          const found = await CartItemDb.findOne({
+            where: {
+              id: cartItem.id,
+            },
+          });
+          !found && await CartItemDb.create({ ...cartItem });
```

Comment:
your DB should be filled with default data using seeds: https://sequelize.org/docs/v6/other-topics/migrations/#creating-the-first-seed